src/components/Security/Users/UserItem.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div :class="{ 'collapsed': collapsed }">
+    <i class="fa fa-caret-down item-toggle" aria-hidden="true" @click="toggleCollapse()"></i>
+
+    <input
+      type="checkbox"
+      class="filled-in"
+      id="checkbox-{{user.id}}"
+      value="{{user.id}}"
+      @click="notifyCheckboxClick"/>
+
+    <label for="checkbox-{{user.id}}">{{user.id}}</label>
+
+    <div class="right actions">
+      <a class="action fa fa-pencil"></a>
+      <dropdown :id="user.id">
+        <li><a @click="deleteUser(user.id)">Delete</a></li>
+      </dropdown>
+    </div>
+
+    <div class="item-content">
+      <pre>{{itemContent}}</pre><div class="profile-list">
```

Comment:
By the way, we had a proper JSON View component in the old BO? Might be better than a `pre`